Atreya/appointments/urls.py

```diff
@@ -3,7 +3,8 @@
 from rest_framework.urlpatterns import format_suffix_patterns
 
 urlpatterns = [
-    path('appointments/<int:appointment_id>',views.appointments),  
+    path('appointments/<int:appointment_id>',views.appointments),
+    path('appointments/get_appointments/<int:appointment_id>',views.get_appointment)  
```

Comment:
what is the value of this second URL? The functionality can be determined from passed in variables